Braintree.xcodeproj/project.pbxproj

```diff
@@ -1876,12 +1878,13 @@
 		9CE5179C282D54030013C740 /* BraintreePayPalNativeCheckoutTests */ = {
 			isa = PBXGroup;
 			children = (
+				BEDB820529B13F9500075AF3 /* BTPayPalNativeCheckoutAccountNonce_Tests.swift */,
+				3DAB593E286CB967003A7BC5 /* BTPayPalNativeCheckoutClient_Tests.swift */,
+				3DAB5932286B40E2003A7BC5 /* BTPayPalNativeCheckoutRequest_Tests.swift */,
```

Comment:
This diff is from sorting the files in this directory by name - which they weren't before